### DISCUSSION TITLE
Chat does not read my notes

### DISCUSSION THREAD

**Discussion by Simonstj** [brianpetro/obsidian-smart-connections#893] [Q&A] [answered]
I have the Chat settings to Open AI using GPT 4 Turbo but when I ask it questions about notes within a Folder I got the following response:

"Based on your notes, I don't have the ability to directly access or view your personal or specific files, including a folder named X My capabilities are limited to providing information and answering questions based on the data I have been trained on. If you need help with specific patterns or files, you might consider providing more details or descriptions, and I can offer guidance or information based on that."

So it looks like Chat is not reading my notes.....can anyone help with why this might be?

**Comment by joelgwebber** [comment]
I've gotten similar responses in some cases, but was able to get it to read a specific note (e.g., to summarize an article) with a prompt like "Please summarize the article [[My Article Name]]". The link reference autocompletes correctly, which makes it a lot easier.

I'm less clear on how to most effectively get it to pick up on a group of articles through embeddings, but ISTR some mention in the docs of it triggering on possessive pronounce like "my" and "mine". I just tried "Can you list articles I have saved in my 'Articles' folder", and it was able to list them correctly.

**Comment by brianpetro** [comment]
In the latest version, it's easy to see if the lookup function is working because it returns a list of notes used as context prior to responding.

Let me know if that helps clear anything up or if there are still issues after upgrading 🌴

**Reply by Simonstj** [reply]
Hey Brian, thanks for reaching out on this. I have upgraded and got a positive result this time. 

However, just wanted to ask your advice.

In a folder I have 100 notes, each is a transcript from forum topics.

If I ask Smart Chat to give me a summary of each topic and what is discussed, it randomly picked 10 notes and completed the task.

Is it possible to get it to iterate over all 100 or is that just too much?

Ideally I'd like to go further and have it do a detailed sumamry of each note, producing a 'manual' of all the wisdom in the content.....but that seems like a far reach....

Would appreciate your thoughts...

Simon

**Reply by dmitriikostin** [reply]
Hi! I have the same problem. What do you mean by upgrading? From what to what?

**Reply by brianpetro** [reply, marked as answer]
@Simonstj the latest version of Smart Chat has a "lookup limit" setting that you can increase above the default 10 🌴

**Reply by Simonstj** [reply]
Awesome - thanks Brian!
